src/3d/shaders/mesh/mesh.geom

```diff
@@ -141,18 +153,27 @@ void main()
         gs_out.worldNormal = normal[0];
         gs_out.magnitude=gs_in[0].magnitude;
         gl_Position = gl_in[0].gl_Position;
+        #ifdef CLIPPING
+            setClipDistance(gs_in[0].worldPosition);
+        #endif
         EmitVertex();
 
         gs_out.worldPosition = gs_in[1].worldPosition;
         gs_out.worldNormal = normal[1];
         gs_out.magnitude=gs_in[1].magnitude;
         gl_Position = gl_in[1].gl_Position;
+        #ifdef CLIPPING
+            setClipDistance(gs_in[1].worldPosition);
+        #endif
         EmitVertex();
 
         gs_out.worldPosition = gs_in[2].worldPosition;
         gs_out.worldNormal = normal[2];
         gs_out.magnitude=gs_in[1].magnitude;
         gl_Position = gl_in[2].gl_Position;
+        #ifdef CLIPPING
+            setClipDistance(gs_in[0].worldPosition);
```

Comment:
is this a typo or is it intentionally 0 instead of 2 ?

Reply:
This was a typo. Fixed.